src/modules/Web3Mail/schemas/timestamp-model.ts

```diff
@@ -0,0 +1,16 @@
+import { Schema, model, models } from 'mongoose';
+
+const timestampSchema = new Schema({
+  type: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  date: {
```

Comment:
lastRunAt

Reply:
Add some fields to help to have some visibility
- lastRunCountErrors 
- lastRunCountSend
- lastRunDuration